crates/swc_ecma_parser/src/error.rs

```diff
@@ -198,7 +198,7 @@ pub enum SyntaxError {
 
     TrailingCommaInsideImport,
 
-    ExportDefaultWithOutFrom,
```

Comment:
Can we keep this variant? Maybe with `#[deprecated]`. Removing a variant is a breaking change.